src/DataCollector/CacheDataCollector.php

```diff
@@ -215,7 +215,7 @@ private function calculateTotalStatistics()
     /**
      * @return callable[] The casters to add to the cloner
      */
-    private function getCasters()
+    protected function getCasters()
```

Comment:
This will override any changes Symfony is making to this file. That might lead to issues when you are using SF 3.2 or higher. 

To be compatible with both 2.8, 3.1, 3.2, 3.3 we should rename this function to any thing else. 

I copied this method from sf 3.3 that is why it has the same name